Update fastlane to avoid vulnerability in mini_magick

yerdua opened the pull request:
### Description

We got a security alert because of the version of `mini_magick` used by the mobile package. This is a dependency of `fastlane 2.127.1`, so can't be updated directly. The fix is included on the master branch of `fastlane` on github, but hasn't made it into a release on RubyGems yet.

For reference, this is the commit where it's fixed: https://github.com/fastlane/fastlane/pull/15042

To avoid waiting for that release to fix the vulnerability here, this PR points the source of `fastlane` at github. This can be switched back once `fastlane`'s next release makes it to [RubyGems](https://rubygems.org/gems/fastlane). This followup task is covered in #53.

### Tested

Tested by locally deploying the app to the store.

m-chrzan commented on packages/celotool/geth_tests/src/lib/utils.ts:
```diff
@@ -379,7 +399,7 @@ export function getHooks(gethConfig: GethTestConfig) {
         validatorIndex++
       }
-      await initAndStartGeth(gethBinaryPath, instance)
+      instance.pid = await initAndStartGeth(gethBinaryPath, instance)
```
Can/should `initAndStartGeth` just set `instance.pid` internally?

nicholasguo replied:
yes

m-chrzan commented on packages/celotool/geth_tests/sync_tests.ts:
```diff
@@ -84,4 +87,28 @@ describe('sync tests', () => {
+      await sleep(20)
+      await hooks.restartInstance(instance)
+      await sleep(100) // wait for round change / resync
+      const address = generateAccountAddressFromPrivateKey(instance.privateKey || '')
```
Why is the `|| ''` necessary?

nicholasguo replied:
Because when first initialized the privateKey field was not included since it is fetched from a store. Typescript thus complains when I don't include `|| ''` because `instance.privateKey` may be undefined

yerdua committed:
Because of a vulnerability in the version of mini_magick that's a
dependency of fastlane, update fastlane. Since the version of
fastlane with the fix isn't released to RubyGems yet, temporarily
point to github as the source for this gem.

yerdua committed:
update TODO with the github issue number

yerdua committed:
Merge branch 'master' into yerdua/use-latest-fastlane-and-minimagick

jmrossy reviewed (dismissed):
LGTM 👍

yerdua committed:
ignore .bundle and vendor

yerdua committed:
Merge branch 'master' into yerdua/use-latest-fastlane-and-minimagick

yerdua committed:
Also make these changes for the verifier

yerdua committed:
Merge branch 'master' into yerdua/use-latest-fastlane-and-minimagick

aaronmgdr commented:
LG from this former rails dev

yerdua committed:
Merge branch 'master' into yerdua/use-latest-fastlane-and-minimagick
